package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "bookstore",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@testing-library/jest-dom": "^5.16.2",
+        "@testing-library/react": "^12.1.2",
+        "@testing-library/user-event": "^13.5.0",
+        "big.js": "^6.1.1",
+        "gh-pages": "^3.2.3",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
+        "react-router-dom": "^6.2.1",
+        "react-scripts": "5.0.0",
+        "web-vitals": "^2.1.4"
```

Comment:
- [ ] according to the requirements you should add React Redux to the dependencies `npm install react-redux`.